Sources/Fullscreen/Consent/Settings/ConsentViewCell.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Copyright © 2018 FINN AS. All rights reserved.
+//
+import UIKit
+
+public enum ConsentState: String {
+    case accepted = "På", withdrawn = "Av"
```

Comment:
Not sure how we usually do it in `FinniversKit`, but I would move the text to localisable strings.

Reply:
I could potentially remove the text and then set the text in the ios-app instead